Contents/Code/youtube_dl_helper.py

```diff
@@ -35,7 +55,10 @@ def process_youtube(url):
     ...
     """
     youtube_dl_params = dict(
-        outmpl='%(id)s.%(ext)s',
+        logger=youtube_logger,
```

Comment:
```suggestion
        logger=plugin_logger,
```

Reply:
Do you mind making all these parameters alphabetical?

Reply:
Oops, I added all of those in nano and didn't run a formatter afterwards. I'll fix that